packages/build/src/babel-plugin-bare-specifiers.ts

```diff
@@ -94,12 +94,7 @@ const maybeResolve = (
     return resolve(specifier, filePath, componentInfo);
   } catch (e) {
     if (!isPathSpecifier(specifier)) {
-      // Don't warn if the specifier was already a path, even though we do
-      // resolve paths, because maybe the user is serving it some other
-      // way.
-      console.warn(
```

Comment:
Would prefer to keep this warning since an unresolved module could be bad. The missing "require" (and now "meta") warnings we should def fix (https://github.com/Polymer/tools/issues/131) since everyone will see them and they are not bad.

Reply:
+1, reverted. I think this now fixes #131